src/api/groups.controller.ts

```diff
@@ -49,6 +49,13 @@ export class GroupsController {
     return session.getGroup(id);
   }
 
+  @Put(':id/groupAdminOnly')
```

Comment:
Let's create a bit more complex structure for that
```suggestion
  @Put(':id/settings/security/info-admin-only')
```